nltk/app/collocations_app.py

```diff
@@ -29,44 +29,46 @@
-                lambda: cess_cat.words(),
-            'English: Brown Corpus':
-                lambda: brown.words(),
-            'English: Brown Corpus (Press)':
-                lambda: brown.words(categories=['news', 'editorial', 'reviews']),
-            'English: Brown Corpus (Religion)':
-                lambda: brown.words(categories='religion'),
-            'English: Brown Corpus (Learned)':
-                lambda: brown.words(categories='learned'),
-            'English: Brown Corpus (Science Fiction)':
-                lambda: brown.words(categories='science_fiction'),
-            'English: Brown Corpus (Romance)':
-                lambda: brown.words(categories='romance'),
-            'English: Brown Corpus (Humor)':
-                lambda: brown.words(categories='humor'),
-            'English: NPS Chat Corpus':
-                lambda: nps_chat.words(),
-            'English: Wall Street Journal Corpus':
-                lambda: treebank.words(),
-            'Chinese: Sinica Corpus':
-                lambda: sinica_treebank.words(),
-            'Dutch: Alpino Corpus':
-                lambda: alpino.words(),
-            'Hindi: Indian Languages Corpus':
-                lambda: indian.words(files='hindi.pos'),
-            'Portuguese: Floresta Corpus (Portugal)':
-                lambda: floresta.words(),
-            'Portuguese: MAC-MORPHO Corpus (Brazil)':
-                lambda: mac_morpho.words(),
-            'Portuguese: Machado Corpus (Brazil)':
-                lambda: machado.words(),
-            'Spanish: CESS-ESP Corpus':
-                lambda: cess_esp.words()
-           }
+    'Catalan: CESS-CAT Corpus':
+    lambda: cess_cat.words(),
+    'English: Brown Corpus':
+    lambda: brown.words(),
+    'English: Brown Corpus (Press)':
+    lambda: brown.words(
+        categories=['news', 'editorial', 'reviews']),
+    'English: Brown Corpus (Religion)':
+    lambda: brown.words(categories='religion'),
+    'English: Brown Corpus (Learned)':
+    lambda: brown.words(categories='learned'),
+    'English: Brown Corpus (Science Fiction)':
+    lambda: brown.words(categories='science_fiction'),
+    'English: Brown Corpus (Romance)':
+    lambda: brown.words(categories='romance'),
+    'English: Brown Corpus (Humor)':
+    lambda: brown.words(categories='humor'),
+    'English: NPS Chat Corpus':
+    lambda: nps_chat.words(),
+    'English: Wall Street Journal Corpus':
+    lambda: treebank.words(),
+    'Chinese: Sinica Corpus':
+    lambda: sinica_treebank.words(),
+    'Dutch: Alpino Corpus':
+    lambda: alpino.words(),
+    'Hindi: Indian Languages Corpus':
+    lambda: indian.words(files='hindi.pos'),
+    'Portuguese: Floresta Corpus (Portugal)':
+    lambda: floresta.words(),
+    'Portuguese: MAC-MORPHO Corpus (Brazil)':
+    lambda: mac_morpho.words(),
+    'Portuguese: Machado Corpus (Brazil)':
+    lambda: machado.words(),
+    'Spanish: CESS-ESP Corpus':
+    lambda: cess_esp.words()
+}
+
```

Comment:
I'm finding the above less readable than the original. So PEP-8 doesn't permit dictionary literals to have their values nested relative to their keys?